### ISSUE TITLE
Not transferred http-agent

### ISSUE THREAD

**Issue by molokovskikh** [eisbehr-/external-ip#1]
I looked source code and I not saw where transferred http-agent.
Anybody know how it do?

For details http://clip2net.com/s/3KZthZR
Was setting config.agent before call extIP. 
My client code like this
```
function _getIp(agent) {  
  var getIp = extIP({  
		agent: agent,  
		timeout: 2000,  
		mode: 'parallel',  
		followRedirect : true,  
     	        maxRedirects   : 10,  
		services       : [  
        	"http://ifconfig.co/x-real-ip",  
        	"http://ifconfig.io/ip"  
    	]  
  });  
  return getIp.get();  
}

**Comment by eisbehr-** [comment]
I think you want to set the browser request http agent, am I right? Then you used the wrong config parameter `agent`, but it should be `userAgent`.

**Comment by dkern** [comment]
Ohh sorry, I missread your question at first. It seem you're right. It's missing the pass of `agent` to the `request` module. I will fix this and write a test for that!

**Comment by dkern** [comment]
I'll fixed this bug with [version 0.3.6](https://github.com/eisbehr-/external-ip/releases/tag/v0.3.6). It's already updated on `npm` too, so you can directly use it.

Thanks for your report!

**Comment by molokovskikh** [comment]
Great man! 
It was operative )
